setup.py

```diff
@@ -23,7 +23,8 @@
     zip_safe=False,
     include_package_data=True,
     platforms="any",
```

Comment:
If you want to maintain Python 2.7 support and remove this `python_requires`, pin `pyngrok>=4.1,<4.2` instead and revert the change in `__init__.py`  (it was introduced as [a breaking change only in `5.0.0`](https://github.com/alexdlaird/pyngrok/blob/main/CHANGELOG.md#500---2020-10-25)).